src/completers/composer.zsh

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env zsh
 
 _fzf_complete_composer() {
-    if [[ $@ = 'composer'* ]]; then
+    local arguments=("${(Q)${(z)@}[@]}")
+    local subcommand=${arguments[2]}
+
+    # The condition is needed to reconsider when the options of `composer` are supported.
```

Comment:
We don't need to note this because it's pretty straightforward that we don't cover the other options here.